nilearn/plotting/surf_plotting.py

```diff
@@ -852,7 +852,7 @@ def plot_surf(
 
     %(vmin)s
     %(vmax)s
-    cbar_vmin, cbar_vmax : float, float, optional
+    cbar_vmin, cbar_vmax : float, float, default=None
```

Comment:
```suggestion
    cbar_vmin, cbar_vmax : float, float, default=None
```

Do you think you could split this in 2?


```
cbar_vmin : :obj:`float` or None, default=None
        Lower bound for the colorbar.
        If None, the value will be set from the data.

cbar_vmin : :obj:`float` or None, default=None
        Lower bound for the colorbar.
        If None, the value will be set from the data.
```